lightningd/jsonrpc.c

```diff
@@ -852,3 +853,15 @@ json_tok_address_scriptpubkey(const tal_t *cxt,
 
 	return ADDRESS_PARSE_UNRECOGNIZED;
 }
+
+bool json_tok_wtx(struct wallet_tx * tx, const char * buffer,
```

Comment:
Other `json_tok_*` functions are usually of the form `bool json_tok_whatever(const char *buffer, const jsmntok_t *tok, struct whatever *output)` and do not call `command_fail` themselves.  This is acceptable, though.